.github/workflows/devbuild.yml

```diff
@@ -227,7 +227,9 @@ jobs:
 
       - name: dependency by homebrew
         run: |
-          # brew update  # No update because it is slow
+          # Reopen for runner image v20231029.1
```

Comment:
This is not related to your code.  It seems that you did not properly rebase and included code from other changes.

Reply:
Thanks for the comment, I have rebased with the latest master.